app/models/scishield_training.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class ScishieldTraining < ApplicationRecord
```

Comment:
Please add a code comment explaining the intent/responsibility of this new class